shadowsocks/tcp_test.go

```diff
@@ -148,3 +150,59 @@ func BenchmarkTCPFindCipherRepeat(b *testing.B) {
 		c.Close()
 	}
 }
+
+var testMetrics = metrics.NewShadowsocksMetrics(nil)
+
+const testTimeout = 200 * time.Millisecond
```

Comment:
Move this and the `testMetrics` to the `probeExpectTimeout` function to limit their scope.

Reply:
done, except moved `testMetrics` to calling function because it causes an error when re-defined